nexus/src/app/mod.rs

```diff
@@ -92,6 +92,9 @@ pub struct Nexus {
 
     /// Operational context used for external request authentication
     opctx_external_authn: OpContext,
+
+    /// Max issue delay for samael crate - used only for testing
+    samael_max_issue_delay: std::sync::Mutex<Option<chrono::Duration>>,
```

Comment:
Can you say more about what this is for?

Reply:
added a comment in 288930f7